flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/FlinkWriteOptions.java

```diff
@@ -60,6 +61,14 @@ private FlinkWriteOptions() {}
   public static final ConfigOption<String> DISTRIBUTION_MODE =
       ConfigOptions.key("distribution-mode").stringType().noDefaultValue();
 
+  public static final ConfigOption<String> RANGE_DISTRIBUTION_STATISTICS_TYPE =
+      ConfigOptions.key("range-distribution-statistics-type")
+          .stringType()
+          .defaultValue(StatisticsType.Auto.name());
+
+  public static final ConfigOption<Double> CLOSE_FILE_COST_WEIGHT_PERCENTAGE =
+      ConfigOptions.key("close-file-cost-weight-percentage").doubleType().defaultValue(0.02d);
```

Comment:
open to feedback on the config name, type, and default value. `0.02` means 2% of close file weight on the target weight per task. it avoids placing more than 50 files in one writer task.

Reply:
Also keeps open 50 writers (which means high memory footprint).

Does `0.02d` mean 2 percent? In this case we can use `close-file-cost-weight`.
Or alternatively if `percentage` is in the name, then the value should be `2`

Reply:
Yes, `0.02d` meant `2%`. I didn't go with the `percentage` in the naming and integer as value in order to get a bit more flexibility like maybe like `0.005d` for `0.5%`.

agree with the comment on naming that is probably not the best. maybe `close-file-cost-weight`. In the doc, we already explained `0.02d` meant `2%`.

Reply:
I renamed this config to `RANGE_DISTRIBUTION_SORT_KEY_BASE_WEIGHT` as I think it is more accurate. added more extensive Javadoc and explanation in the doc. hope that it is more clear to users.